src/entrypoints/my-change-url.ts

```diff
@@ -76,13 +76,13 @@ class MyChangeUrl extends LitElement {
           : ""}
 
         <my-url-input
-          .value=${this._instanceUrl}
+          .values=${this._instanceUrl}
           @value-changed=${this._handleUrlChanged}
         ></my-url-input>
 
         ${this._error ? html`<div class="error">${this._error}</div>` : ""}
 
-        <p>Note: This URL is only stored in your browser.</p>
+        <p>Note: The URLs is only stored in your browser.</p>
```

Comment:
The URLs feels weird. Let's keep it singular. As the note applies to each row.
```suggestion
        <p>Note: The URL is only stored in your browser.</p>
```

Reply:
Correct the grammar here.
<p>Note: The URLs is only stored in your browser.</p>
to <p>Note: The **URLs are** only stored in your browser.</p>